mysql/requirements.in

```diff
@@ -1,5 +1,5 @@
 cachetools==3.1.1
 cryptography==3.3.2; python_version < '3.0'
-cryptography==3.4.8; python_version > "3.0"
+cryptography==35.0.0; python_version > "3.0"
```

Comment:
From the agent build: `snowflake-connector-python 2.6.0 has requirement cryptography<4.0.0,>=2.5.0, but you have cryptography 35.0.0.` I think it's safe to put this in `ignored_deps` and remove this change